In [ ]:
import pandas as pd
import numpy as np
import scipy.stats as stats

In [1]:
# Find Z-Scores  for 44  Z = (X - µ) / σ 
Z= (44 - 38)/6
Z

1.0

through z table we got 84.13 % so People above 44 age = 100 - 84.13 =  15.87%  that is  63  out of 400

In [2]:
#similarly for
## Find Z-Scores  for  38  Z = (X - µ) / σ 
Z= (38 - 38)/6
Z

0.0

through z table we got  50% so People between 38 & 44  age = 84.13 - 50 = 34.13 %  137 out of 400 hence Therefore, More employees at the processing center are older than 44 than between 38 and 44 is FALSE .

In [3]:
#Z score for 30 
z=(30 - 38)/6 
z

-1.3333333333333333

 -1.33  =  9.15 % = 36 out of 400 hence
B. Therefore, A training program for employees under the age of 30 at the center would be expected to attract about 36 employees - TRUE.